early_tpm.c

```diff
@@ -98,6 +54,8 @@ u8 *tpmb_reserve(struct tpmbuff *b)
 
 void tpmb_free(struct tpmbuff *b)
 {
+	memset(b->head, 0, b->len);
```

Comment:
LGTM is complaining that this is an implicit declaration.

Reply:
Yes, I know, this is also for the previous PR so I'll need to fix it there and rebase.

Reply:
Done